AnnoyancesFilter/Widgets/sections/widgets.txt

```diff
@@ -1828,6 +1828,8 @@ zebratechnologies.force.com##.zbrbot-container
 !
 !   SECTION: Uncategorized
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/180486 - share button
+nowfoods.com###block-sdsocialshareblock
```

Comment:
I think it should be added to `Social Media filter`.
Could you please move it somewhere here (without a link to issue) - https://github.com/AdguardTeam/AdguardFilters/blob/master/SocialFilter/sections/specific.txt#L77